juniper_official/Solutions/Pfe-wedge-detection/eachip-pattern-set.rule

```diff
@@ -0,0 +1,489 @@
+                }
+                key-fields fpc_number;
+            }
+            pattern EACHIP_EVENT014 {
+                event-id EACHIP_EVENT014;
+                filter "fpc%{NUMBER:fpc_number}.*MQSS.*FI.*Cell underflow at the state stage.*Cell behind reorder window.*Stream.*Count";
+                description "The \"Cell underflow at the state stage\" message reports a cell arriving late at the egress PFE. The is due to the fabric cells arriving late on the egress PFE behind the reorder window.";
+                constant syslog-trigger-color {
+                    value red;
+                }
+                constant syslog-trigger-message {
+                    value "No issue if single error.  Contact JTAC if count > 100/sec.";
+                }
+                key-fields fpc_number;
+            }
+            pattern EACHIP_EVENT015 {
+                event-id EACHIP_EVENT015;
+                filter "fpc%{NUMBER:fpc_number}.*MQSS.*FI.*Cell jump drop error";
+                description "The \"Child drop error\" message reports cell is dropped during allocation in the fabric input reorder engine.";
+                constant syslog-trigger-color {
+                    value red;
+                }
+                constant syslog-trigger-message {
+                    value "Investigate originating FPC";
+                }
+                key-fields fpc_number;
+            }
+            pattern EACHIP_EVENT016 {
+                event-id EACHIP_EVENT016;
+                filter "fpc%{NUMBER:fpc_number}.*EA.*idmem_slice.*Corrected single bit ECC error";
+                description "The \"Corrected single bit ECC error\" message is likely caused due to transient Hardware. Onchip IDMEM memory is ECC protected. If the number of single-bit ECC correction reaches 32 event within 24 hours, an CMERROR Minor Alarm Single-bit ECC error is reported. The counter will be reset every day change. Single bit ECC errors will be corrected.";
+                constant syslog-trigger-color {
+                    value red;
+                }
+                constant syslog-trigger-message {
+                    value "If single log, no action.  If multiple errors at same location after minor alarm is raised, contact JTAC.";
+                }                       
+                key-fields fpc_number;
+            }
+            pattern EACHIP_EVENT017 {
+                event-id EACHIP_EVENT017;
+                filter "fpc%{NUMBER:fpc_number}.*MQSS.*DRD.*CMD FSM state error";
+                description "The \"CMD FSM state error\" message reports that the Forwarding State Machine of the reorder queue received an unexpected value. This indicates the lookup microcode within LUSS is sending an incorrect command parcel to the MQSS Center Chip.";
+                constant syslog-trigger-color {
+                    value red;
+                }
+                constant syslog-trigger-message {
```

Comment:
Please add a count function to track the number of occurrences.

Reply:
@ConstrainedRandomVar , added trigger based on count